src/Package/Impl/StatusBar/VsStatusBar.cs

```diff
@@ -45,10 +45,15 @@ internal sealed class VsStatusBar : IStatusBar {
         }
 
         #region IStatusBar
-        public IDisposable AddItem(UIElement element) {
+        public IDisposable AddItem(Func<UIElement> itemFactory, object dataContext) {
```

Comment:
I'd rather leave it as it was and disable tests for now. This adds complexity that is required only for test itself